modules/effects/arrow-effects/src/main/kotlin/arrow/effects/internal/IOPlatform.kt

```diff
@@ -0,0 +1,21 @@
+package arrow.effects.internal
+
+internal object IOPlatform {
```

Comment:
We don't really need this object because IO will never work outside the JVM. Merge it in another file if you want.

Reply:
> never work outside the JVM

We never plan to support IO on js?

Reply:
These functions are used from a couple of places so not sure where to place them. I've renamed the file to `ErrorUtils` for now, following the same style than `CancelUtils`.